index.html

```diff
@@ -820,13 +820,27 @@ <h3>JsonWebKey</h3>
               </dd>
               <dt><dfn class="lint-ignore">publicKeyJwk</dfn></dt>
               <dd>
+                <p>
 The `publicKeyJwk` property is REQUIRED, and its value MUST
 be a JSON Web Key that conforms to [[RFC7517]].
 It is RECOMMENDED that verification methods that use
 JWKs [[RFC7517]] to represent their <a>public keys</a> use the value of `kid` as
 their fragment identifier. It is RECOMMENDED that JWK `kid` values be set to
 the public key fingerprint [[RFC7638]]. See the first key in the example below
 for an instance of a public key with a compound key identifier.
+                </p>
+                <p>
+As specified in <a data-cite="RFC7517#section-4.4">Section 4.4 of the JWK specification</a>,
+the `alg` property identifies the algorithm intended for use with the public key, 
+and is included to prevent security issues that can arise when using the same
+key with multiple algorithms. As specified in <a data-cite="RFC7518#section-6.2.1.1">
+Section 6.2.1.1 of the JWA specification</a>, describing a key using an elliptic curve, 
+the `crv` property is used to identify the particular curve type of the public key. 
+As specified in <a data-cite="RFC7515#section-4.1.4">Section 4.1.4 of the JWS specification</a>,
+the `kid` property is a hint used to help discover the key; if present, the `kid` value SHOULD
+match, or be included in, the `id` property of the encapsulating `JsonWebKey` object,
+as part of the path, query, or fragment of the URL.
```

Comment:
```suggestion
the OPTIONAL `alg` property identifies the algorithm intended for use with the public key, 
and SHOULD be included to prevent security issues that can arise when using the same
key with multiple algorithms. As specified in <a data-cite="RFC7518#section-6.2.1.1">
Section 6.2.1.1 of the JWA specification</a>, describing a key using an elliptic curve, 
the REQUIRED `crv` property is used to identify the particular curve type of the public key. 
As specified in <a data-cite="RFC7515#section-4.1.4">Section 4.1.4 of the JWS specification</a>,
the OPTIONAL `kid` property is a hint used to help discover the key; if present, the `kid` value SHOULD
match, or be included in, the `id` property of the encapsulating `JsonWebKey` object,
as part of the path, query, or fragment of the URL.
```

Reply:
I suggest we briefly discuss this on an upcoming call 
cc: @selfissued @brentzundel